bundle/InfoCollection/Form/Type/HoneypotType.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\InfoCollection\Form\Type;
+
+use Ibexa\ContentForms\Form\Type\FieldType\TextLineFieldType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class HoneypotType extends AbstractType
+{
+    public function getBlockPrefix(): string
+    {
+        return 'information_collection';
```

Comment:
Ovo treba bit identifikator vrzan uz honeypot (npr `honeypot`). Ovo je previse genericki.

Reply:
Cilj ove promjene je postići isto handleanje kroz blokove form teme kao što ga imaju ostali fieldovi, generirani iz Contenta. Plus ovo jest pod `InfoCollection` namespaceom.

Reply:
To mi je jasno, no ovo se koristi za override field templatea u form temi, dakle ako bi overrideao template za honeypot field, block u Twigu bi se zvao `information_collection_widget` sto nema smisla.

Da ne govorim o tome da se ovaj type potencijalno moze koristit van info collectora, sad kad je decouplan.

Reply:
Zapravo ima smisla, taj field ne smije biti po ni čemu poseban osim što je skriven preko CSS-a.
Budući da je cijela komponenta toliko zapetljana, mislim da nema previše smisla raditi neko generičko rješenje.
Glavni trik je zapravo u CSS-u.

Reply:
Ne bih se slozio. To sto field ne treba bit poseban po nicemo ne invalidira cinjenicu da nekad treba dodat nekakav markup (mozda prije, mozda poslije fielda, ne nuzno na samom fieldu). Ovo, kao i drugi komentar su samo coding standardi iz Symfonya koje ne bi bilo lose pratit radi konzistentnosti i na nikoji nacin ne umanjuju sposobnost samog fielda da izgleda ili se ponasa kao ostali fieldovi na info collector formi.

Oba ova slucaja bi bila ekvivalentna tome da npr neki symfony servis imena MyService ima identifier u DI tagu `sarma`. Nema nikakvog efekta na funkcionalnost, al nije ni cisto ni konzistentno.

Reply:
Stvar je u kaskadi renderiranja kroz Twig blokove u form temi, moramo imati istu strukturu da bi se tema aplicirala na sva polja jednako. Ajmo radije sutra to uživo prokomentirati, biti će lakše nego da tu tipkamo.